platform/marvell-armhf/sonic-platform-et6448m/inband_mgmt.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+#inband_mgmt
+
+inband_mgmt(){
+
+# The intent of this sequence is to ensure 12c bus enumeration order by
+# controlling the order in which the various i2c device drivers are
+# loaded. Hence the drivers are unloaded and then reloaded in the prescribed
+# order.
+# NOTE: In the nokia platform the following sequence is performed by the Nokia
+# platform service init script and thus should not be performed here
+ grep ^onie_platform /host/machine.conf 2>/dev/null | grep nokia >/dev/null
+ if [ $? != 0 ]; then
+     rmmod i2c-dev
+     rmmod i2c_mux_gpio
+     rmmod i2c_mv64xxx
+     modprobe i2c_mv64xxx
+     modprobe i2c-dev
+     modprobe i2c_mux_gpio
+     sleep 60
```

Comment:
>sleep 60 [](start = 5, length = 8)

In general I am worried about the magic number sleep in all scripts. Do you want to achieve some collaboration between them? Is there any risk in future platforms?

Reply:
There is no use to start inband mgmt script before entire system is ready. So we have added 60sec sleep
This is specific only to this platform. Not applicable for other platforms.

Reply:
Thank you for the info. So my concern is "entire system is ready" < 60s, any risk here? If risk happens, what is the impact?

---
In reply to: [526620478](https://github.com/Azure/sonic-buildimage/pull/5749#discussion_r526620478) [](ancestors = 526620478)

Reply:
There is no risk in that case. In-band mgmt port will be brought up after that.